wmfdata/utils.py

```diff
@@ -9,6 +10,19 @@
 import pandas as pd
 import requests
 
+"""
+Global wmfdata.utils.log logger.  Usage:
+
+from wmfdata.utils import log
+log.info("my message")
+"""
+log = logging.getLogger('wmfdata')
+log.setLevel(logging.INFO)
+formatter = logging.Formatter(fmt='%(asctime)s %(levelname)s %(name)s: %(message)s', datefmt='%Y-%m-%dT%H:%M:%S')
+handler = logging.StreamHandler(sys.stdout)
```

Comment:
I'd use `sys.stderr` instead. In a Jupyter notebook, that gets displayed with a light red background, so it's clearly distinguishable as a log message. The downside is the red makes it seem more like a warning, but to me it's worth it (I use the print_err function below for a number of informational things).

Reply:
Too keep this PR simple, I'll just use print_err instead of a logger then.

We can add better logging in a future PR.